### PR TITLE
Sockets - Maria 

### PR DESCRIPTION
**Pull request by MariaWissler**
# Task List
Congratulations! You're submitting your assignment!

## Comprehension Questions
Question | Answer
:------------- | :-------------
Describe in your own words what the Model is doing in Rails |  Model keeps track of the Database and thats were the business logic happens 
Describe in your own words what the Controller is doing in Rails | Controller has the methods our app needs to function, interacts with the user, and the models 
Describe in your own words what the View is doing in Rails |  Shows the info to the user 
Describe an edge-case controller test you wrote |  Just creating a random task id number for the for it respond to an un existent id
What is the purpose of using strong params? (i.e. the params method in the controller) | A feature to ensure users cannot update sensitive model attributes that shouldn't be modified 
How are Rails migrations related to Rails models? |  They facilitate life, add or modify rails models 
Describe one area of Rails that are still unclear on |  Most of the basic parts are clear, however I need to work more in my html skills and how to have a more decent view presented, clear but does require more practice

### CODE REVIEW

File: app/views/tasks/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<h1>YOUR TASKS ʕ•́ᴥ•̀ʔっ</h1>
```
Comment (dHelmgren) [inline app/views/tasks/index.html.erb:1]: Your task list is unbelievably cute, in part thanks to this good bear.

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Rails.application.routes.draw do
+  root "tasks#index"
+
+  get "/tasks", to: "tasks#index", as: "tasks"
+  get "/tasks/new", to: "tasks#new", as: "new_task"
+  post "/tasks", to: "tasks#create"
+
+  get "/tasks/:id", to: "tasks#show", as: "task"
+  get "tasks/:id/edit", to: "tasks#edit", as: "edit_task"
```
Comment (dHelmgren) [inline config/routes.rb:11]: I would have liked to see you using `resources` here to dry things up!

**Comment by dHelmgren** [comment]
# Task List
## What We're Looking For

Feature | Feedback
:------------- | :-------------
**Baseline** |
Appropriate Git Usage with no extraneous files checked in | yes
Answered comprehension questions | yes
Successfully handles: Index, Show | yes
Index & Show tests pass | yes
Successfully handles: New, Create | yes
New & Create tests pass | yes
Successfully handles: Edit, Update | yes
Tests for Edit & Update test valid & invalid task ids | yes
Successfully handles: Destroy, Task Complete | yes
Tests for Destroy & Task Complete include tests for valid and invalid task ids | yes
Routes follow RESTful conventions | yes
Uses named routes (like `_path`) | yes
**Overall** | Good job on Task List! It functions as expected. You hit all of the learning goals: using partials, using strong params, and following Rails conventions.
